index.js

```diff
@@ -0,0 +1,50 @@
+function validatePassword(valid) {
+  if (passwordLength(valid) === true && passwordLowerCase(valid) === true && passwordUpperCase(valid) === true && passwordNumericValue(valid) === true && passwordSpecialCharacter(valid) === true) {
+    return true
+  }
+  else {
+    return false
+  }
+}
+function passwordLength(valid) {
+  if (valid.length >= 8) {
+    return true
+  }
+  else {
+    return false
+  }
+}
+function passwordLowerCase(valid) {
```

Comment:
Functions should always be verbs since we name them what they do like `validatePassword`. Predicate functions should use `is` or `has` as their verbs since they return true/false. `hasLowerCase()`